### ISSUE TITLE
Group Managment

### ISSUE THREAD

**Issue by claasklar** [claasklar/Signal-Desktop#19]
It is not possible to manage groups in the Signal-Desktop client. Managing groups includes:
- Changing group name
- Leaving a group
- Add a group member
- Create a group

This is already a feature request in the [forums](https://community.signalusers.org/t/ability-to-create-groups-in-desktop-app/850) and in this [issue](https://github.com/signalapp/Signal-Desktop/issues/530). Apparently the signal team wants to realize this feature. Therefore I think, implementing all above mentioned use cases, would be a waste of time. The point I get the most annoyed about is, that you cannot leave a group. So leaving a group could be a good first step to resolve this issue, until the feature is implemented in upstream.

**Comment by alexandrughinea** [comment]
I can't event see the groups listed in the desktop version. Is this something related or ...?

**Comment by claasklar** [comment]
I don't think so. Feel free to open an new issue. It would be great to know, if the problem exists on both primary and secondary devices. They introduced version 2 groups and I was not able to test synchronization with those groups yet. As you can imagine it's a little cumbersome to test groups, when you cannot create them.
